### ISSUE TITLE
[CAL-913] public booking page

### ISSUE THREAD

**Issue by Jaibles** [calcom/cal.com#6698]
design a better way to handle long text descriptions

![CleanShot 2023-01-25 at 15.11.38@2x.png](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/d204a1bb-baaa-4251-8bb1-820ea27eb532/f62a12b2-18eb-4d3d-92a3-f37015286d5c)

[https://cal.com/instaprice/15min-demo](https://cal.com/instaprice/15min-demo)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-913](https://linear.app/calcom/issue/CAL-913/public-booking-page)</sub>

**Comment by PeerRich** [comment]
where is the design for this?

**Comment by Jaibles** [comment]
meant to assign me not Jeroen - it needs design.

**Comment by angel8au** [comment]
Hi @Jaibles ,

I was wondering if there is any documentation on how to contribute to design issues. Also, I wanted to express my interest in contributing to tasks related to design.

**Comment by Jaibles** [comment]
Hi @angel8au there are currently no guidelines for this. It's definitely something I've thought about. Let me think about how we might allow for this and come back to you.

**Comment by angel8au** [comment]
I started looking through the Figma files for the booking screen, and I found one where I was able to identify the error. In the original design (fFgma file), the column containing the appointment information is 280px wide, while in the live version it's only 230px wide, which makes the text look somewhat ugly. My recommendation is to increase the width to 280px, as proposed in the original design.

![image](https://user-images.githubusercontent.com/37971517/219275898-90ab80b9-e1ec-476a-974f-019ef04e9106.png)
